### ISSUE TITLE
The build scheme file (.xcscheme) made by generate-xcode-support can have unescaped spaces in the paths

### ISSUE THREAD

**Issue by mikekreuzer** [stackotter/swift-bundler#49]
Sonoma 14.4.1, Xcode 15.3, Swift 5.10, swift-bundler 2.0.3

## To see it:

```bash
swift bundler create HelloBusted --template SwiftUI && \
cd HelloBusted && \
swift bundler generate-xcode-support && \
open Package.swift
```

Hit the big blue button to build & run... & it'll fail with the only semi useful error: Executable Path is a Directory.

## Cause:
BuildAction & PathRunnable in HelloBusted.xcscheme both have paths to directories in ~/Library/Application Support/, & the space in Application Support isn't escaped.

## Workaround:
Edit the file to escape any spaces in paths, eg <code>Application\ Support</code>,  restart xCode & all's well. Application Support's likely to be there for everyone.

## Fix:
Add another escape in Sources/swift-bundler/Bundler/XcodeSupportGenerator.swift I guess.

Extremely cool project, had it recommended to me yesterday. Thank you!

**Comment by stackotter** [comment]
Thanks for reporting this issue! Must've been introduced by a newish version of Xcode, cause I can reproduce the issue but last time I tested Xcode support it worked (which was probably 6 months ago at this point).

I've added an extra escape which resolves the issue on my end, hopefully that resolves it for you too 🤞

> Extremely cool project, had it recommended to me yesterday. Thank you!

Thanks! I'm always interested how people find out about Swift Bundler, did you get recommended it by a colleague?

**Comment by mikekreuzer** [comment]
Fixed in v3.0.0! Quick turn around, ta. 😄

Found you via social media, Mastodon, which if you haven't heard of it's a Twitter clone. (Clone is probably not doing it justice.) I was [complaining about rolling packages by hand](https://mstdn.social/@mike_k/112158472089288708) & got the recommendation.

**Comment by stackotter** [comment]
> Fixed in v3.0.0!

Woohoo!

> I was [complaining about rolling packages by hand](https://mstdn.social/@mike_k/112158472089288708) & got the recommendation.

Ah, good old Sven spreading the word 😅
